src/video/x11/SDL_x11window.c

```diff
@@ -716,9 +716,11 @@ X11_GetWindowTitle(_THIS, Window xwindow)
                     0L, 8192L, False, XA_STRING, &real_type, &real_format,
                     &items_read, &items_left, &propdata);
         if (status == Success && propdata) {
+            SDL_LogError(SDL_LOG_CATEGORY_VIDEO, "Failed to convert _WM_NAME title expecting UTF8! Title: %s", title);
```

Comment:
In general we shouldn't add log messages to release code. You can make that LogDebug, or #ifdef some_debug_flag. If this is a fatal error, then we should return the message via SDL_SetError() and return an error code.